lib/core/form/components/widgets/radio-buttons/radio-buttons.widget.spec.ts

```diff
@@ -211,6 +189,33 @@ describe('RadioButtonsWidgetComponent', () => {
                     expect(element.querySelector('#radio-id-opt-1-input')).toBeNull();
                 });
             }));
+
+            describe('and radioButton is readonly', () => {
+
+                beforeEach(async(() => {
+                    radioButtonWidget.field.readOnly = true;
+                    fixture.detectChanges();
+                }));
+
+                it('should show radio buttons disabled', async(() => {
+                    expect(element.querySelector('.adf-readonly')).toBeDefined();
+                    expect(element.querySelector('.adf-readonly')).not.toBeNull();
+                }));
+
+                describe('and a value is selected', () => {
+
+                    beforeEach(async(() => {
+                        radioButtonWidget.field.value = restOption[0].id;
+                        fixture.detectChanges();
+                    }));
+
+                    it('should check the selected value', async(() => {
+                        expect(element.querySelector('.mat-radio-checked')).toBeDefined();
+                        expect(element.querySelector('.mat-radio-checked')).not.toBeNull();
```

Comment:
If you already check later for a fix value you don't need also those toBeDefined and not.toBeNull before